examples/simulation/Tutorial2/macros/CMakeLists.txt

```
@@ -20,52 +20,54 @@ add_test(NAME ex_sim_tutorial2
 set_tests_properties(ex_sim_tutorial2 PROPERTIES
   TIMEOUT ${maxTestTime}
   PASS_REGULAR_EXPRESSION "Macro finished successfully"
+  FIXTURES_SETUP fixtures.ex_sim_tutorial2_sim
 )
 
 add_test(NAME ex_sim_tutorial2_create_digis
          COMMAND ${CMAKE_CURRENT_BINARY_DIR}/create_digis.sh)
 set_tests_properties(ex_sim_tutorial2_create_digis PROPERTIES
-  DEPENDS ex_sim_tutorial2
+  FIXTURES_REQUIRED fixtures.ex_sim_tutorial2_sim
   TIMEOUT ${maxTestTime}
   PASS_REGULAR_EXPRESSION "Macro finished successfully"
+  FIXTURES_SETUP fixtures.ex_sim_tutorial2_digi
 )
 
 add_test(NAME ex_sim_tutorial2_read_digis
          COMMAND ${CMAKE_CURRENT_BINARY_DIR}/read_digis.sh)
 set_tests_properties(ex_sim_tutorial2_read_digis PROPERTIES
-  DEPENDS ex_sim_tutorial2_create_digis
+  FIXTURES_REQUIRED fixtures.ex_sim_tutorial2_digi
   TIMEOUT ${maxTestTime}
   PASS_REGULAR_EXPRESSION "Macro finished successfully"
 )
 
 add_test(NAME ex_sim_tutorial2_run_background
          COMMAND ${CMAKE_CURRENT_BINARY_DIR}/run_background.sh)
 set_tests_properties(ex_sim_tutorial2_run_background PROPERTIES
-  DEPENDS ex_sim_tutorial2_read_digis
   TIMEOUT ${maxTestTime}
   PASS_REGULAR_EXPRESSION "Macro finished successfully"
+  FIXTURES_SETUP fixtures.ex_sim_tutorial2_back
 )
 
 add_test(NAME ex_sim_tutorial2_run_signal1
          COMMAND ${CMAKE_CURRENT_BINARY_DIR}/run_signal.sh 1 10)
 set_tests_properties(ex_sim_tutorial2_run_signal1 PROPERTIES
-  DEPENDS ex_sim_tutorial2_run_background
   TIMEOUT ${maxTestTime}
   PASS_REGULAR_EXPRESSION "Macro finished successfully"
+  FIXTURES_SETUP fixtures.ex_sim_tutorial2_sig1
 )
 
 add_test(NAME ex_sim_tutorial2_run_signal2
          COMMAND ${CMAKE_CURRENT_BINARY_DIR}/run_signal.sh 2 20)
 set_tests_properties(ex_sim_tutorial2_run_signal2 PROPERTIES
-  DEPENDS ex_sim_tutorial2_run_signal1
   TIMEOUT ${maxTestTime}
   PASS_REGULAR_EXPRESSION "Macro finished successfully"
+  FIXTURES_SETUP fixtures.ex_sim_tutorial2_sig2
 )
 
 add_test(NAME ex_sim_tutorial2_create_digis_mixed
          COMMAND ${CMAKE_CURRENT_BINARY_DIR}/create_digis_mixed.sh)
 set_tests_properties(ex_sim_tutorial2_create_digis_mixed PROPERTIES
-  DEPENDS ex_sim_tutorial2_run_signal2
+  FIXTURES_REQUIRED "fixtures.ex_sim_tutorial2_back;fixtures.ex_sim_tutorial2_sim1;fixtures.ex_sim_tutorial2_sim2"
```

```suggestion
  FIXTURES_REQUIRED "fixtures.ex_sim_tutorial2_sim;fixtures.ex_sim_tutorial2_back;fixtures.ex_sim_tutorial2_sig1;fixtures.ex_sim_tutorial2_sig2"
```

https://cdash.gsi.de/testDetails.php?test=12792197&build=356328
```
Error in <TFile::TFile>: file /Users/alfaci/alfaci/workspace/FairRootGroup_FairRoot_PR-1284/build/examples/simulation/Tutorial2/macros/./tutorial2_pions.mc_p2.000_t0_n10.sg1.root does not exist
```